### PR TITLE
Remove useless file and remove template variable declarations

### PR DESCRIPTION
**Pull request by kurtw**
An index.html file was created in /initiatives/ by mistake and some template variables are set incorrectly. This removes them.

@KimberlyMunoz @anselmbradford @jimmynotjim

### CODE REVIEW

File: offices/_vars-offices.html [hunk 1]
```diff
@@ -2,9 +2,6 @@
 {% set query = queries.initiative %}
 {% set initiatives = query.search_with_url_arguments(size=6, filter_has_parent="false", filter_related_office=office.slug) %}
 
-{% set tag = "foia" %}
-{% set activity_type = ["cfpb_report"] %}
```
Comment (dpford) [inline offices/_vars-offices.html]: It looks like _single.html uses the variable activity_type.  Are we even doing activity feeds for offices at this point?

Comment (kurtw) [inline offices/_vars-offices.html]: The activity type for the FOIA page is being set in the /offices/foia-requests/index.html file. I don't know what the decision is on whether or not we're using them in Offices. It was just in there. Also, @KimberlyMunoz and I discussed moving the activity_type variable to the cms so we don't have to create an entire folder/file just to set it.

Comment (dpford) [inline offices/_vars-offices.html]: :+1:

**Comment by anselmbradford** [comment]
LGTM :+1:

**Pull request merged**